pkg/filter/plugins/plugins.go

```diff
@@ -165,7 +189,7 @@ func getAPIFilterFuncsWithPluginsGroup(plu *config.PluginsInUse) []context.Filte
 
 	// found with group name
 	for _, groupName := range plu.GroupNames {
-		pwfMap, found := groupWithPluginsMap[groupName]
+		pwfMap, found := groupCache[groupName]
 		if found {
```

Comment:
`	if !found {
		continue
	}
	for _, pwf := range pwfMap {
		tmpMap[pwf.Name] = pwf.fn
	}
`